examples/local/run.sh

```diff
@@ -1 +1 @@
-train --config config.yaml --output output
+train --config config.yaml --output output --no-model-restoring --train
```

Comment:
I think we have a bug here.. the first train should be main:
`main --config config.yaml --output output --no-model-restoring --train`